### ISSUE TITLE
"Skill level" bar in presets

### ISSUE THREAD

**Issue by elreymu** [theonlydude/RandomMetroidSolver#18]
Because there are some premade presets such as noob,casual,regular,veteran,speedrunner and master which have (in my opinion) pretty accurate perceived difficulty settings and basically range from least skilled to most skilled in that order.
a skill level estimator bar could be made comparing the current custom preset with all the settings of the premade presets.
Similar to the difficulty estimator bar, ranging from noob to master.
it can help to tweak your own custom preset, it would give you an idea whether you're overdoing it or going too easy with what you actually want to experience, nice to have something to compare it to.

Basically it shows how close or similar your custom preset is to the premade ones as you're making or editing your own.
(taking techniques, hellruns, hard rooms and bosses settings into account for the comparison, basically every setting relevant)

**Comment by theonlydude** [comment]
![image](https://user-images.githubusercontent.com/3090510/44811678-899dbd00-abd5-11e8-8a48-f021feb64bfc.png)
Something like that ?

**Comment by theonlydude** [comment]
Hello,
The skill level bar has been pushed to production.
We're also displaying some stats about the presets now.
